test/plain.jl

```diff
@@ -1546,7 +1546,6 @@ end
     @test !HDF5.ishdf5(fn1) # check that a non-hdf5 file retuns false
     @test !HDF5.ishdf5(fn2) # checks that a file that does not exist returns false
 
-    @test_throws ErrorException h5write(fn1, "x", 1) # non hdf5 file throws
```

Comment:
Same as the other comment. Comment this out or consider checking to see if the file is actually writable.